docs/Basic-Guides.md

```diff
@@ -0,0 +1,152 @@
+# Basic Guide
+
+This guide will show you how to use a pretrained model in an example Unity environment, and show you how to train the model yourself.
+
+If you are not familiar with the [Unity Engine](https://unity3d.com/unity),
+we highly recommend the [Roll-a-ball tutorial](https://unity3d.com/learn/tutorials/s/roll-ball-tutorial) to learn all the basic concepts of Unity. 
+
+## Setting up ML-Agents within Unity
+
+In order to use ML-Agents within Unity, you need to change some Unity settings first. Also TensorflowSharp plugin is needed for you to use pretrained model within Unity. 
+
```

Comment:
Can you put link for TensorFlowSharp and capitalize the f of TensorFlowSharp

Reply:
Will change.